framework_tvb/tvb/core/services/import_service.py

```diff
@@ -180,17 +175,24 @@ def import_project_structure(self, uploaded, user_id):
             if os.path.exists(temp_folder):
                 shutil.rmtree(temp_folder)
 
-    def _import_projects_from_folder(self, temp_folder):
+    def _import_project_from_folder(self, temp_folder):
         """
         Process each project from the uploaded pack, to extract names.
         """
-        project_roots = []
+        temp_project_path = None
         for root, _, files in os.walk(temp_folder):
             if StorageInterface.TVB_PROJECT_FILE in files:
-                project_roots.append(root)
+                temp_project_path = root
+                break
 
-        for temp_project_path in project_roots:
+        if temp_project_path is not None:
             update_manager = ProjectUpdateManager(temp_project_path)
+
+            if update_manager.checked_version < 3:
+                raise ImportException('Importing projects with older versions is unsupported! Please import the '
```

Comment:
Inform which is the minimum version supported: "Importing projects with versions older than 3 is not supported in TVB 2...launch the current version of TVB in order to upgrade this project!" ?